lms/djangoapps/certificates/tests/tests.py

```diff
@@ -90,7 +90,8 @@ def test_certificate_info_for_user(self, allow_certificate, whitelisted, grade,
         {'allow_certificate': True, 'whitelisted': True, 'grade': None, 'output': ['Y', 'Y', 'honor']},
         {'allow_certificate': True, 'whitelisted': False, 'grade': 0.9, 'output': ['Y', 'Y', 'honor']},
         {'allow_certificate': False, 'whitelisted': True, 'grade': 0.8, 'output': ['N', 'Y', 'honor']},
-        {'allow_certificate': False, 'whitelisted': None, 'grade': 0.8, 'output': ['N', 'Y', 'honor']}
+        {'allow_certificate': False, 'whitelisted': None, 'grade': 0.8, 'output': ['N', 'Y', 'honor']},
+        {'allow_certificate': True, 'whitelisted': None, 'grade': None, 'output': ['Y', 'Y', 'honor']}
```

Comment:
1. How to read the outpu there?
2. For the following `N` is it `N` for Eligibility? 
```{'allow_certificate': False, 'whitelisted': False, 'grade': None, 'output': ['N', 'Y', 'honor']},```

Reply:
`['Certificate Eligible', 'Certificate Delivered', 'Certificate Type'] = ['Y', 'Y', 'honor']`

Reply:
please add `{'allow_certificate': False, 'whitelisted': True, 'grade': None` scenrio as well.

Reply:
@awaisdar001 I have added this scenario.